.github/workflows/release.yml

```diff
@@ -24,9 +24,6 @@ jobs:
       published_packages: ${{ steps.changesets.outputs.publishedPackages }}
       published: ${{ steps.changesets.outputs.published }}
     steps:
-      - name: 🛑 Cancel Previous Runs
-        uses: styfle/cancel-workflow-action@0.11.0
-
```

Comment:
I didn't add `concurrency` to the release workflow since we probably want any started release to finish?  For example if we start a `pre.0` release and then quickly fix a bug and kick off a `pre.1` - we would want them both to complete and publish to npm instead of leaving a gap in the prereleases?

1. It should never overlap since it runs once every 24 hours